From 1cd810bbfcdf410388f9459119e9a2354dad4545 Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Fri, 13 Sep 2013 11:20:52 +0800
Subject: [PATCH 250/602] ENGR00279092 wifi-display:prevent double processing
 for wlan0 and p2p0

Here it is obselete, wlan0 and p2p0 share the same radio type, so system will
have two copy of wifidatatracker to double process some broadcasts which will
make wifi display not stable in some cases.This change refer to what lge do
in commit 2b37050c17cf5c2cc58c5c4eda7447aa8423e48f.

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 arm2_6dq/overlay/frameworks/base/core/res/res/values/config.xml       | 1 -
 arm2_6sl/overlay/frameworks/base/core/res/res/values/config.xml       | 1 -
 evk_6sl/overlay/frameworks/base/core/res/res/values/config.xml        | 1 -
 hdmidongle_6dq/overlay/frameworks/base/core/res/res/values/config.xml | 1 -
 imx53_smd/overlay/frameworks/base/core/res/res/values/config.xml      | 1 -
 sabreauto_6q/overlay/frameworks/base/core/res/res/values/config.xml   | 1 -
 sabresd_6dq/overlay/frameworks/base/core/res/res/values/config.xml    | 1 -
 7 files changed, 7 deletions(-)

diff --git a/arm2_6dq/overlay/frameworks/base/core/res/res/values/config.xml b/arm2_6dq/overlay/frameworks/base/core/res/res/values/config.xml
index a7177f0..bfadd5c 100755
--- a/arm2_6dq/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/arm2_6dq/overlay/frameworks/base/core/res/res/values/config.xml
@@ -79,7 +79,6 @@
         <item>"mobile_fota,10,0,2,60000,true"</item>
         <item>"mobile_ims,11,0,2,60000,true"</item>
         <item>"mobile_cbs,12,0,2,60000,true"</item>
-        <item>"wifi_p2p,13,1,0,-1,true"</item>
         <item>"ethernet,9,9,0,-1,true"</item>
     </string-array>
     <!-- Base "touch slop" value used by ViewConfiguration as a
diff --git a/arm2_6sl/overlay/frameworks/base/core/res/res/values/config.xml b/arm2_6sl/overlay/frameworks/base/core/res/res/values/config.xml
index 79a5123..bd2f67b 100755
--- a/arm2_6sl/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/arm2_6sl/overlay/frameworks/base/core/res/res/values/config.xml
@@ -79,7 +79,6 @@
         <item>"mobile_fota,10,0,2,60000,true"</item>
         <item>"mobile_ims,11,0,2,60000,true"</item>
         <item>"mobile_cbs,12,0,2,60000,true"</item>
-        <item>"wifi_p2p,13,1,0,-1,true"</item>
         <item>"ethernet,9,9,0,-1,true"</item>
     </string-array>
     <!-- Base "touch slop" value used by ViewConfiguration as a
diff --git a/evk_6sl/overlay/frameworks/base/core/res/res/values/config.xml b/evk_6sl/overlay/frameworks/base/core/res/res/values/config.xml
index a585088..226b65d 100644
--- a/evk_6sl/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/evk_6sl/overlay/frameworks/base/core/res/res/values/config.xml
@@ -79,7 +79,6 @@
         <item>"mobile_fota,10,0,3,60000,true"</item>
         <item>"mobile_ims,11,0,3,60000,true"</item>
         <item>"mobile_cbs,12,0,3,60000,true"</item>
-        <item>"wifi_p2p,13,1,0,-1,true"</item>
         <item>"ethernet,9,9,2,-1,true"</item>
     </string-array>
 
diff --git a/hdmidongle_6dq/overlay/frameworks/base/core/res/res/values/config.xml b/hdmidongle_6dq/overlay/frameworks/base/core/res/res/values/config.xml
index a3f163b..e25b1f1 100755
--- a/hdmidongle_6dq/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/hdmidongle_6dq/overlay/frameworks/base/core/res/res/values/config.xml
@@ -48,7 +48,6 @@
         <item>"mobile_fota,10,0,3,60000,true"</item>
         <item>"mobile_ims,11,0,3,60000,true"</item>
         <item>"mobile_cbs,12,0,3,60000,true"</item>
-        <item>"wifi_p2p,13,1,0,-1,true"</item>
         <item>"ethernet,9,9,2,-1,true"</item>
     </string-array>
 
diff --git a/imx53_smd/overlay/frameworks/base/core/res/res/values/config.xml b/imx53_smd/overlay/frameworks/base/core/res/res/values/config.xml
index cc77b94..a59d08e 100755
--- a/imx53_smd/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/imx53_smd/overlay/frameworks/base/core/res/res/values/config.xml
@@ -68,7 +68,6 @@
         <item>"mobile_fota,10,0,3,60000,true"</item>
         <item>"mobile_ims,11,0,3,60000,true"</item>
         <item>"mobile_cbs,12,0,3,60000,true"</item>
-        <item>"wifi_p2p,13,1,0,-1,true"</item>
         <item>"bluetooth,7,7,2,-1,true"</item>
         <item>"ethernet,9,9,2,-1,true"</item>
     </string-array>
diff --git a/sabreauto_6q/overlay/frameworks/base/core/res/res/values/config.xml b/sabreauto_6q/overlay/frameworks/base/core/res/res/values/config.xml
index e2931b8..72b2cf0 100755
--- a/sabreauto_6q/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/sabreauto_6q/overlay/frameworks/base/core/res/res/values/config.xml
@@ -78,7 +78,6 @@
         <item>"mobile_fota,10,0,3,60000,true"</item>
         <item>"mobile_ims,11,0,3,60000,true"</item>
         <item>"mobile_cbs,12,0,3,60000,true"</item>
-        <item>"wifi_p2p,13,1,0,-1,true"</item>
         <item>"bluetooth,7,7,2,-1,true"</item>
         <item>"ethernet,9,9,2,-1,true"</item>
     </string-array>
diff --git a/sabresd_6dq/overlay/frameworks/base/core/res/res/values/config.xml b/sabresd_6dq/overlay/frameworks/base/core/res/res/values/config.xml
index fb500e0..17a281f 100755
--- a/sabresd_6dq/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/sabresd_6dq/overlay/frameworks/base/core/res/res/values/config.xml
@@ -69,7 +69,6 @@
         <item>"mobile_fota,10,0,3,60000,true"</item>
         <item>"mobile_ims,11,0,3,60000,true"</item>
         <item>"mobile_cbs,12,0,3,60000,true"</item>
-        <item>"wifi_p2p,13,1,0,-1,true"</item>
         <item>"bluetooth,7,7,2,-1,true"</item>
         <item>"ethernet,9,9,2,-1,true"</item>
     </string-array>
-- 
1.8.0

